features/step_definitions/basic_steps.rb

```diff
@@ -50,6 +50,11 @@
   visit edit_recipe_path(recipe)
 end
 
+Given("I am on the {string} page") do |recipe_title|
```

Comment:
You should consider specifically state that you want to be on the `show` or `details` page for a recipe

Reply:
:+1: